kuma/wiki/helpers.py

```diff
@@ -277,3 +279,15 @@ def document_zone_management_links(user, document):
                                   args=(zone.id,))
 
     return links
+
+
+@register.filter
+def absolutify(url, site=None):
```

Comment:
Could you check if `url` is just a slash and set it to an empty string here? So we don't have to do `absolutify('')` but `absolutify('/')`?